pkgs/development/ocaml-modules/hxd/default.nix

```
@@ -5,22 +5,17 @@
 
 buildDunePackage rec {
   pname = "hxd";
-  version = "0.3.1";
+  version = "0.3.2";
 
   useDune2 = true;
 
   minimumOCamlVersion = "4.06";
```

```suggestion
  minimumOCamlVersion = "4.08";
```
https://github.com/ocaml/opam-repository/blob/master/packages/hxd/hxd.0.3.2/opam#L19

---

And spell it minim**al**.